library/gen_x12_837i.inc.php

```diff
@@ -0,0 +1,1308 @@
+<?php
+/**
+ * X12 837I
+ *
+ * @package OpenEMR
+ * @link    http://www.open-emr.org
+ * @author  Jerry Padgett <sjpadgett@gmail.com>
+ * @copyright Copyright (c) 2017 Jerry Padgett <sjpadgett@gmail.com>
+ * @license https://www.gnu.org/licenses/agpl-3.0.en.html GNU Affero General Public License 3
+ */
```

Comment:
Place the following line (after you rebase to the most recent codebase) and ensure you have a blank line before and after this line:
``````
use OpenEMR\Billing\Claim;
``````